infra.bs

```diff
@@ -7,7 +7,11 @@ Abstract: The Infra Standard aims to define the fundamental concepts upon which
 Translation: ja https://triple-underscore.github.io/infra-ja.html
 </pre>
 
-<pre class="anchors">
+<pre class=link-defaults>
+spec:infra; type:dfn; text:string
+</pre>
```

Comment:
This can be removed again tomorrow I suspect because of https://github.com/w3c/csswg-drafts/commit/ed9468f9f168a5ed6c7d2ef2cea6134bf4a63cd5.